app/models/verification_attempt.rb

```diff
@@ -0,0 +1,7 @@
+class VerificationAttempt < ApplicationRecord
+  belongs_to :client
+
+  # I'm guessing we will need to link somehow to Active_storage to define what selfie and photo_id are?
+  has_one_attached :selfie
+  has_one_attached :photo_id
```

Comment:
we'll def want to update this to call it "photo_identification" -- photo_id will cause confusion, thinking its an id reference to a table called photos /model named Photo 😬